src/server/scripts/EasternKingdoms/Karazhan/boss_prince_malchezaar.cpp

```diff
@@ -378,9 +368,43 @@ struct npc_malchezaar_axe : public ScriptedAI
         TaskScheduler _scheduler;
 };
 
+// 30843 - Enfeeble
+class spell_malchezaar_enfeeble : public SpellScript
+{
+    PrepareSpellScript(spell_malchezaar_enfeeble);
+
+    bool Load()
+    {
+        return GetCaster()->ToCreature();
+    }
+
+    void FilterTargets(std::list<WorldObject*>& targets)
+    {
+        uint8 maxSize = 5;
+        Unit* caster = GetCaster();
```

Comment:
can be null?

Reply:
caster can never be null during spell cast

Reply:
ah this is during spellcast, okay